### PR TITLE
alignment problem in text view fixed

### PR DESCRIPTION
**Pull request by mahikaw**
The text view looks like this now. Fixes issue #227 

device-2016-12-20-220105

### CODE REVIEW

**Comment by anubhakushwaha** [comment]
@mahikaw Please it is a kind request,do not send a PR without getting an approval for the same from any of the mentor on your proposed issue.Thanks!

**Comment by mahikaw** [comment]
@anubhakushwaha, I think this is a very obvious issue as the misalignment looks really bad and it is better to fix it asap. 
I, on the other hand am waiting for an approval from the mentors on other issues which might not be intended. :smile:

**Comment by mahikaw** [comment]
@anubhakushwaha, I know I need to get approval from the mentors, however, this is a genuine 'bug' and needed fixing asap. Thanks, nevertheless :smile:

**Comment by chhavip** [comment]
@mahikaw , as pointed by @anubhakushwaha , do not send PR without prior approval if you wish for them to be reviewed. As for this, please remove .idea and .iml files as pointed in other PR and re-submit fore review.

**Comment by mahikaw** [comment]
@chhavip I have been waiting for approval on obvious issues like #224 and #222 part(b), however this was  an obvious bug, but I will still take care of this in future. Thanks for reviewing :smiley:

**Comment by chhavip** [comment]
@mahikaw You need to understand that mentors are here on volunteer basis and with full time work of their own. It will do you good to be respectful to their time and show patience as other students/collaborators do. Also fix the PRs as indicated.

**Comment by mahikaw** [comment]
@chhavip, I totally respect your time and other work commitments. I just really wanted to see this app in a better shape. I assure you that I will be waiting for approvals and consents in future before working on any issue. 
Also, I was having some trouble with the changes you asked for in my previous PRs and it would be great if you could help me with that.
Thanks a lot. :smile:

**Comment by anubhakushwaha** [comment]
@mahikaw Yeah sure,just commit without adding these files.

**Comment by mahikaw** [comment]
@chhavip I'm not able to squash the last two commits because of the merge that you created for gitignore. However, no extra files are updated now so this can be merged. Thanks.

**Comment by mahikaw** [comment]
@chhavip I'm not able to squash the last two commits because of the merge that you created for gitignore. However, no extra files are updated now so this can be merged. Thanks.

File: PowerUp/app/src/main/res/layout/completed_game.xml [hunk 1]
```diff
@@ -12,6 +12,8 @@
         android:layout_height="@dimen/completed_game_height"
         android:text="@string/completed_game"
         android:textColor="@android:color/black"
-        android:textSize="@dimen/game_text_size" />
+        android:textSize="@dimen/game_text_size"
+        android:layout_marginLeft="10dp"
```
Comment (chhavip) [inline PowerUp/app/src/main/res/layout/completed_game.xml]: Do not use hard coded values, please add in dimes.xml

Comment (mahikaw) [inline PowerUp/app/src/main/res/layout/completed_game.xml]: okay sure.

**Comment by mahikaw** [comment]
Okay, i'm closing this one then

**Pull request closed without merging**